src/twisted/test/test_defer.py

```diff
@@ -1516,6 +1516,23 @@ def test_errbackWithNoArgsNoDebug(self) -> None:
         self.assertEqual([], localz)
         self.assertEqual([], globalz)
 
+    def test_errbackAwaitTwiceNoDebug(self) -> None:
+        defer.setDebugging(False)
+        d: Deferred[None] = Deferred()
+
+        class MyError(Exception):
+            pass
+
+        async def c() -> None:
+            for _ in range(2):
+                with self.assertRaises(MyError):
+                    await d
+
+        d2 = Deferred.fromCoroutine(c())
+        d.errback(MyError())
+        self.assertIsNone(d._debugInfo)
```

Comment:
for some reason: `defer.fail(MyError())._debugInfo` is always truthy see https://twistedmatrix.com/trac/ticket/10223#comment:1